gcn/notices/km3net/alert.example.json

```
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/km3net/alert.schema.json",
+  "messenger": "Neutrino",
+  "mission": "KM3NeT",
+  "instrument": "ORCA024",
+  "pipeline": "exceptional_evt_orca",
+  "alert_tense": "current",
+  "alert_type": "initial",
+  "record_number": 1,
+  "alert_datetime": "2024-09-01T12:01:00.00Z",
+  "additional_info": "Observation from single pipeline. Analysis tuned to select 1 event/month in average. Track only analysis w/ Up-going selection.",
+  "id": "1",
+  "event_name": ["KM3-240901A"],
```

```suggestion
  "event_name": ["KM3-240901A"],
```

We have updated the event schema (https://github.com/nasa-gcn/gcn-schema/pull/275), so that array is not must for single event_name.
Please update the changes accordingly.